src/sickle.ts

```diff
@@ -146,12 +168,14 @@ class Annotator {
    * Returns empty string if there is no existing annotation.
    */
   private existingClosureAnnotation(p: ts.PropertyDeclaration | ts.ParameterDeclaration) {
-    let comments = ts.getLeadingCommentRanges(this.file.text, p.pos);
+    let text = p.getFullText();
+    let comments = ts.getLeadingCommentRanges(text, 0);
```

Comment:
For some reason the old code was unable to get comments unless I did this first.
I think this is harmless though so I don't mind the change.